### ISSUE TITLE
[BUG] Accessibility - Web app has improper focus order

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5051]
**Describe the bug**
When utilizing Talkback on Android Chrome browser for https://api-dev.rokwire.illinois.edu/illinois/ There is little to no focus indication, and when some is present, it is not in the order which makes sense. 

**To Reproduce**
1. Turn Talkback on
2. Go to '.https://api-dev.rokwire.illinois.edu/illinois/..'
3. Click on '..Maps..'
4. Continue swiping to see what is on Maps page but it goes to Wallet instead. 

**Expected behavior**
Focus should land on title of form/page first to let the user know, then it should have the help text be keyboard navigable and able to receive focus for assistive technology users to understand what this form does and what the inputs are. 

**Screenshots**
See Video. 

https://github.com/user-attachments/assets/ae1f83fb-6aae-4a8a-8ccc-b0ff5a4cbe83

Desktop (please complete the following information):

OS: Windows 10
Browser Chrome
Version 135.0.7049.85
Smartphone (please complete the following information):

Device: Google Pixel 8a
OS: Android 15
Browser Chrome
Version 135.0.7049

**Additional context**

**Comment by TodorBachvarov** [comment]
@acovher2 I've made some accessibility improvements for the new Maps UI in version 7.3.13. I think this is fixed.
